README.md

```diff
@@ -1,110 +1,46 @@
+4. When the tutorial opens, work through the various steps.
 
-![Three Step Didact Tutorial Example](./images/helloJS-didact-demo-APR-2020.gif)
+![Run the HelloWorld Example](./images/run-js-example.gif)
 
-## Accessing Didact in the workspace. 
+## Writing your first Didact tutorial
 
-With Didact installed, there are three ways to open a Didact window.
+1. Create a new Markdown or AsciiDoc file with the extension `.didact.md` or `.didact.adoc`.
+2. Write some text about the action your user will accomplish. 
+3. Start a new Didact link:
+* In Markdown, type `[Open a new terminal]()`, put the cursor between the parentheses `()` and press `Ctrl+Space`. 
+* In AsciiDoc, type `link:[Open a new terminal]`, put the cursor after `link:`, and press `Ctrl+Space`.
+4. Select `Start new Didact command link`.
+5. Choose a command from the hundreds VS Code has available (like `workbench.action.terminal.new`) and press `Enter`.
+6. Press `Ctrl+Alt+V` to view your new Didact tutorial and click the link you created! Done!
 
-* To access Didact, access the Command Palette (`View->Command Palette`, `Ctrl+Shift+P`, or `F1`) and type **Didact**. Select `Open Didact` and it will open with the default Didact file specified in the Settings.
-* To open a local Didact file in the workspace directly, Right-click on the Didact file (`*.didact.md` or `*.didact.adoc`) and select **Start Didact Tutorial from File** from the context menu.
-
-In addition, you now have the `Didact Tutorials` view. 
-
-![Didact Tutorials View](./images/didact-view-with-popup.png)
-
-If you right-click on a tutorial (such as "Didact Demo"), you see a `Start Didact Tutorial` menu, which opens the Didact window or changes the current Didact window to the selected tutorial. These entries are provided by registering new tutorial name/tutorial uri/category combinations with Didact's tutorial registry.
-
-*NEW!* You can also simply mouse over a registered tutorial and hit the 'play' button (triangle) to launch it in the Didact window.
-
-![One Click Open a Didact Tutorial](./images/didact-tutorial-start-button.gif)
-
-## Didact Settings 
-
-To change the default Didact file, access the settings (`File->Preferences->Settings`), type **Didact** and set the `Didact: Default Url`. This can be any valid URL to a Didact formatted Markdown file, including `http://`, `https://`, or `file://` URLs.
-
-To turn on generic Didact completion notifications (off by default), access the settings (`File->Preferences->Settings`), type **Didact** and set the `Didact: Disable Notifications` checkbox. If unchecked, additional notifications will occur when Didact actions are activated in the IDE.
-
-To have Didact open a window with the default Didact file on startup, access the settings (`File->Preferences->Settings`), type **Didact** and set the `Didact: Open Default Tutorial At Startup` checkbox.
-
-![Didact Settings](./images/settings.jpg)
```

Comment:
Can you point where in the wiki there is the information about available settings?

Reply:
good catch. will add to wiki

Reply:
https://github.com/redhat-developer/vscode-didact/wiki/Didact-Settings